static/usage.html

```diff
@@ -618,16 +618,16 @@ <h2><a href="#web-browsing">Web browsing</a></h2>
                 a WebView implementation (GeckoView is not a WebView implementation), so it has to be
                 used alongside the Chromium-based WebView rather than instead of Chromium, which means
                 having the remote attack surface of two separate browser engines instead of only one.
-                Firefox / Gecko also bypass or cripple a fair bit of the upstream and GrapheneOS
+                Firefox/Gecko also bypasses or cripples a fair bit of the upstream and GrapheneOS
                 hardening work for apps. Worst of all, Firefox does not have internal sandboxing
-                on Android. This is despite the fact that Chromium semantic sandbox layer on
+                on Android. Even though Chromium semantic sandbox layer on
```

Comment:
Unsure, but if this change is made wouldn't it be better to combine the two sentences? This way, separating the two sentences seems awkward to me.

For example: "Worst of all, Firefox does not have internal sandboxing on Android even though [the] Chromium semantic sandbox layer on Android implemented via the OS..."

Reply:
Yeah, it is better indeed.